app-text/xapers/xapers-0.9.0.ebuild

```diff
@@ -18,6 +18,8 @@ LICENSE="GPL-3"
 SLOT="0"
 KEYWORDS="~amd64"
 
+DISTUTILS_USE_SETUPTOOLS=rdepend
```

Comment:
This is a '@PRE-INHERIT` variable, you can see a mention if it in [eclass docs](https://devmanual.gentoo.org/eclass-reference/distutils-r1.eclass/index.html) or by running pkgcheck:
```
MisplacedEclassVar: version 0.9.0: invalid pre-inherit placement, line 21: 'DISTUTILS_USE_SETUPTOOLS=rdepend'
```

...**however**, if you look at the top, you'll see it's already set. It may be good to re-read what the QA notice said about this.